src/test/data/jvm/ksp/CoffeeAppService.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2018 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package src.test.data.jvm.ksp;
```

Comment:
This package looks incorrect.

Reply:
The other tests like `kapt` have `tests.smoke.kapt`. Should i go with the same here `tests.smoke.ksp`

Reply:
Should be fine without then